modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -3,10 +3,25 @@
 
 Because the monitoring agent requires a SOCKS5 proxy, you need two proxy types if you use Anypoint Monitoring.
 
+[NOTE]
+Starting Runtime Fabric version 2.6.11, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version less than 2.6.22.
```

Comment:
Is it <2.6.11> or <2.6.22> ? Both versions should be the April 9 release version

Reply:
We will need to update the version here once we have the RC.